ui/src/theme.js

```diff
@@ -297,7 +297,7 @@ const createTheme = isDarkTheme => {
       },
       MuiPickersToolbarText: {
         toolbarTxt: {
-          color: THEME.PRIMARY_TEXT_DARK,
+          color: THEME.PRIMARY_TEXT_DIMMED,
```

Comment:
We won't be able to use `THEME.PRIMARY_TEXT_DIMMED` on the light theme outside the toolbar. This is because the toolbar has a fixed background color. I think it's okay to hard-code the value here. We're also doing this in the theme file so it shouldn't be too bad.

Reply:
Got it. I updated the PR.